diff --git a/node_modules/@manypkg/cli/dist/cli.cjs.dev.js b/node_modules/@manypkg/cli/dist/cli.cjs.dev.js
index b09af02..4a6f0a3 100644
--- a/node_modules/@manypkg/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@manypkg/cli/dist/cli.cjs.dev.js
@@ -187,6 +187,12 @@ var EXTERNAL_MISMATCH = makeCheck({
 
         if (deps) {
           for (var depName in deps) {
+            if (["css-loader"].includes(depName)) {
+              continue
+            }
+            if (workspace.packageJson.name === '@blitzjs/next' && ["source-map", "recast", "chalk"].includes(depName)) {
+              continue
+            }
             var range = deps[depName];
             var highestRange = highestExternalRanges.get(depName);
 
@@ -351,6 +357,9 @@ var INVALID_DEV_AND_PEER_DEPENDENCY_RELATIONSHIP = makeCheck({
 
     if (peerDeps) {
       for (var depName in peerDeps) {
+        if (['fibers', 'node-sass', 'react', 'react-dom', 'next', 'sass', '@mdx-js/loader', '@mdx-js/react', 'webpack'].includes(depName)) {
+          continue
+        }
         if (!devDeps[depName]) {
           var highestRanges = getHighestExternalRanges(allWorkspaces);
           var idealDevVersion = highestRanges.get(depName);
