diff --git a/node_modules/release/bin/release.js b/node_modules/release/bin/release.js
index a5ab985..ce9475a 100755
--- a/node_modules/release/bin/release.js
+++ b/node_modules/release/bin/release.js
@@ -52,25 +52,37 @@ let githubConnection;
 let repoDetails;
 
 const changeTypes = [
-	{
-		handle: 'major',
-		name: 'Major Change',
-		pluralName: 'Major Changes',
-		description: 'incompatible API change'
-	},
-	{
-		handle: 'minor',
-		name: 'Minor Change',
-		pluralName: 'Minor Changes',
-		description: 'backwards-compatible functionality'
-	},
-	{
-		handle: 'patch',
-		name: 'Patch',
-		pluralName: 'Patches',
-		description: 'backwards-compatible bug fix'
-	}
-];
+  {
+    handle: 'major',
+    name: 'Major Change',
+    pluralName: '🔥 Major Changes',
+    description: 'incompatible API change',
+  },
+  {
+    handle: 'minor',
+    name: 'Minor Change',
+    pluralName: '🚀 Minor Changes',
+    description: 'backwards-compatible functionality',
+  },
+  {
+    handle: 'patch',
+    name: 'Patch',
+    pluralName: '🐞 Patches',
+    description: 'backwards-compatible bug fix',
+  },
+  {
+    handle: 'example',
+    name: 'Example App',
+    pluralName: '👀 Changes to Example Apps',
+    description: 'change to example app',
+  },
+  {
+    handle: 'meta',
+    name: 'Meta',
+    pluralName: 'Internal Meta Changes',
+    description: 'internal meta change',
+  },
+]
 
 const getReleaseURL = (release, edit = false) => {
 	if (!release || !release.html_url) {
